nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/merge/RecordBinManager.java

```diff
@@ -245,6 +245,23 @@ public int completeFullEnoughBins() throws IOException {
         return handleCompletedBins(RecordBin::isFullEnough, "Bin is full enough");
     }
 
+    public boolean isAnyBinFullEnough() {
```

Comment:
Do you plan to use this somewhere?  I didn't detect any uses.

Reply:
Nope. Refactored some stuff and didn't need it. Looks like I just overlooked removing the method.